demo/meeting/src/containers/MeetingRoster.tsx

```diff
@@ -32,7 +33,11 @@ const MeetingRoster = () => {
   const attendeeItems = attendees.map((attendee: any) => {
     const { chimeAttendeeId } = attendee || {};
     return (
-      <RosterAttendee key={chimeAttendeeId} attendeeId={chimeAttendeeId} />
+      <RosterAttendee
+        key={chimeAttendeeId}
+        attendeeId={chimeAttendeeId}
+        menu={<VideoStreamMetrics attendeeId={chimeAttendeeId} />}
```

Comment:
Can we change to use a stat icon or something instead of menu? And can we update it to only be visible if the attendee turns on video?

Reply:
Unfortunately  we have to use menu here since it is the defined value in https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/components/ui/Roster/RosterCell/index.tsx#L113, which is used to show the popover menu for roster cells.
Yes, I will make it only visible if attendee turns on video.

Reply:
Actually I haveed tried to hide the icon in roster and I found it is not doable to add this logic in the demo level. Due to the hook rule: https://reactjs.org/docs/hooks-rules.html and the way using to render RosterAttendee.  https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/demo/meeting/src/containers/MeetingRoster.tsx#L32-L37 I need to get attendeeId here in order to get isVideoEnabled and in order to get isVideoEnabled, I have to call useAttendeeStatus hook. But I cannot dynamically pass in attendeeId in the hook in this attendeeItems function in meeting roster.

Reply:
Sorry I saw this now, but since this is something for video only can we show specific to video tile and no in roster?

Reply:
Talking to Trung offline. Using a wrapper for RosterAttendee and we can pass in attendeeId as props and do checks there to show the popover menu

Reply:
@devalevenkatesh I don't think we need to change the mechanism on how to show the stats. This is a design decision we already settled down with Jaime. What I am gonna do is to add a wrapper for RosterAttendee and we can pass in attendeeId as props and do checks there to show the popover menu